packages/dd-trace/src/plugins/util/inferred_proxy.js

```diff
@@ -0,0 +1,95 @@
+const tags = require('../../../../../ext/tags')
+
+const RESOURCE_NAME = tags.RESOURCE_NAME
+const HTTP_ROUTE = tags.HTTP_ROUTE
+const SPAN_KIND = tags.SPAN_KIND
+const HTTP_URL = tags.HTTP_URL
+const HTTP_METHOD = tags.HTTP_METHOD
+
+const PROXY_HEADER_NAME = 'x-dd-proxy-name'
```

Comment:
Was there an RFC for these header named? Seems like a shame they aren't combined into 1 header.

Reply:
From our tests, it seems like it's not possible to combine these values from AWS into 1 header, so our hands are pretty much tied and we're stuck with using separate headers.